### ISSUE TITLE
Sidebar disappears when linking to an anchor or using the table of contents on WCAG 2.2 web page if Reduced Motion is turned on

### ISSUE THREAD

**Issue by amberhinds** [w3c/tr-design#365]
This issue pertains to https://www.w3.org/TR/WCAG22/ please let me know if I am not posting it in the right place.

In multiple browsers, after clicking one of the anchor links in the table of contents or if linking directly to the page with an anchor appended on the URL, the table of contents sidebar disappears and cannot be restored. 

Here is a [video that shows the behavior I am seeing.](https://app.screencastify.com/v3/watch/7HK2pxAi3ZWNrbhvvui5) 

There are no console errors. 

When I inspect the code, this appears to be a bug related to the prefers reduced motion CSS:

```
@media (prefers-reduced-motion: reduce) {
    body.toc-sidebar #toc {
        position: absolute;
        overflow: initial;
        bottom: unset;
    }
}
```

I do have reduced motion turned on in my OS. If I turn off reduced motion, the sidebar stays visible when interacting with it. If I inspect the code and disable the position:absolute in the reduced motion media query above, it works as expected.

**Comment by kfranqueiro** [comment]
Thanks for the video illustrating the issue, and sorry for the delay in response.

I've attempted to reproduce this issue (also on Firefox / Mac with Reduce motion enabled), but I'm not encountering the sidebar hiding when I follow links in the table of contents. As far as I can tell, the styles identified in the video are still present.

Are you still able to reproduce this issue in the latest version of Firefox?

**Comment by kfranqueiro** [comment]
Update: Kevin and I were both able to reproduce this issue in various Chromium-based browsers.

**Comment by kfranqueiro** [comment]
I've transferred this issue from the wcag repository to the tr-design repository, where the CSS in question resides.

It appears these styles were intentionally added in #294 to address #287, where the independently-scrolling sidebar caused migraines.

The implementation of #294 against prefers-reduced-motion, however, causes surprising effects for people who might not expect this behavior: having the OS's reduced motion setting turned on makes the table of contents seem to "disappear" in the majority of cases when clicking an item within it, as the table of contents is far shorter than the length of the document. (And in all cases, the table of contents' position will shift.)

Moreover, the "collapse / pop out sidebar" link technically still works with reduced motion enabled, but may seem like it doesn't when a user ends up in this scrolled-down state and doesn't realize the table of contents has scrolled up out of the current viewport.

Relatedly, a "jump to table of contents" link appears specifically when you collapse the sidebar... but this does not appear in the case of reduced motion where the table of contents is simply scrolled out of view, which may also give the impression of seeming broken.

I am unsure whether or not the "collapse / pop out sidebar" link already existed prior to #294, and am wondering whether that in itself could be a more predictable on-demand solution to #287, without #294?

Or, on the other hand, is this explanation sufficient and this should be considered not a bug (although it will likely continue to surprise/confuse people)?
